test/test_interpolations.jl

```diff
@@ -80,6 +80,23 @@ for interpolation in (
             @test __outward_normal(coords, facenodes) ≈ normal
         end
     end
+    # regression for https://github.com/Ferrite-FEM/Ferrite.jl/issues/520
+    interpolation_type = typeof(interpolation).name.wrapper
```

Comment:
is there a better way to extract the type of the interpolation, e.g. `Lagrange`, `Serendipity` etc.?

Reply:
You can check `interpolation isa Lagrange` instead

Reply:
ideally this should be checked for non-lagrangian interpolations, too. Since in the codebase  `DiscontinuousLagrange`, `CrouzeixRaviart`, `BubbleEnrichedLagrange` is already introduced

Reply:
Also I will include H(div) and H(curl) interpolation soon, because our group need them.

Reply:
> is there a better way to extract the type of the interpolation, e.g. Lagrange, Serendipity etc.?

@koehlerson : I noted down this trick with ref to this thread (used it in testing as well), but recently I saw 
```julia
@inline basetypeof(x::T) where T = basetypeof(T)
@generated function basetypeof(::Type{T}) where T
    if T isa Union
        T
    else
        getfield(parentmodule(T), nameof(T))
    end
end
```
from https://github.com/rafaqz/DimensionalData.jl/blob/3087e3e445ddb57e3f02630d6077cff585a132ac/src/LookupArrays/utils.jl#L73-L80